lib/katello/engine.rb

```diff
@@ -132,9 +132,17 @@ class Engine < ::Rails::Engine
 
     config.to_prepare do
       Katello::CandlepinEventListener.client_factory = proc do
+        # These settings are pulled from settings.yaml intentionally
+        # And are not meant to be switched to the database Settings
```

Comment:
Perhaps also add a reason?
```suggestion
        # And are not meant to be switched to the database Settings
        # because that can lead to issues with the database pool size
```